pywbem_mock/_wbemconnection_mock.py

```diff
@@ -35,8 +35,10 @@
 import traceback
 import re
 from xml.dom import minidom
+from collections import Counter
```

Comment:
`collections.Counter` was introduced in Pyhton 2.7, and thus prevents that this PR can be rolled back into 0.15 (which still needs to support Python 2.6).

Reply:
DONE - I have updated PR #1955 to use `backport_collections.Clounter` for Python 2.6.